runtime/server/instances.go

```diff
@@ -49,7 +49,7 @@ func (s *Server) CreateInstance(ctx context.Context, req *runtimev1.CreateInstan
 	// Check OLAP connection
 	olap, err := drivers.Open(inst.OLAPDriver, inst.OLAPDSN)
 	if err != nil {
-		return nil, status.Error(codes.InvalidArgument, err.Error())
+		return nil, status.Error(codes.InvalidArgument, err.Error()+" (Arguments: "+inst.OLAPDriver+" "+inst.OLAPDSN+")")
```

Comment:
1. Don't add a DSN to an error/log message, it could be bad for security
2. Make it a formatted error instead, like `status.Errorf(codes.InvalidArgument, "could not connect to driver '%s': %w", inst.OLAPDriver, err)`